exasol_script_languages_developer_sandbox/lib/vm_bucket.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from exasol_script_languages_developer_sandbox.lib.aws_access import AwsAccess
+from exasol_script_languages_developer_sandbox.lib.render_template import render_template
+
+STACK_NAME = "VM-Bucket"
```

Comment:
Hm, thinking about if we should make this name more unique to the project

Reply:
changed to `VM-SLC-Bucket`